### PR TITLE
fix(JWTManager): Mark private key as sensitive

### PR DESCRIPTION
**Pull request by provokateurin**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@ More information is available in the External sites documentation.]]></descripti
 	<repair-steps>
 		<post-migration>
 			<step>OCA\External\Migration\CopyDefaultIcons</step>
+			<step>OCA\External\Migration\JWTTokenPrivateKeySensitive</step>
```
Comment (julien-nc) [inline appinfo/info.xml:47]: So you're going to run this on each upgrade. Don't you wanna run it only once?
I did that in a migration step in a bunch of apps.

Comment (provokateurin) [reply appinfo/info.xml:47]: Abusing the database migrations to do this feels wrong to me.
Executing this repair step is extremely cheap, so I think this is fine.

**Review by julien-nc — APPROVED** [review]

**Pull request merged**